lib/test-handler/describe.js

```diff
@@ -20,31 +20,30 @@ const {
  */
 const recordTestResults = dataResult => {
     const { testResults, relativePath, perfStats } = dataResult;
-    const name = getTestKey(getTestName(dataResult));
+    const name = getTestName(dataResult);
+    const key = getTestKey(name);
+
+    if (!key) {
+        reportException(name);
+        return {};
+    }
 
     return testResults.reduce((accumulator, test) => {
-        const { title, status, duration } = test;
```

Comment:
why you removed "duration"?

Reply:
actually i don't remember, but i think that it's useless or not applicable for describe strategy